### ISSUE TITLE
Error: command 'sts.vscode-spring-boot.enableClasspathListening' not found

### ISSUE THREAD

**Issue by Tyriar** [spring-projects/sts4#939]
This error is showing up pretty high in our error telemetry:

```
Error: command 'sts.vscode-spring-boot.enableClasspathListening' not found
at b.k (out/vs/workbench/workbench.desktop.main.js:1551:2618)
at b.executeCommand (out/vs/workbench/workbench.desktop.main.js:1551:2534)
```

You may have renamed but forgot an instance of sts -> Pivotal?

**Comment by BoykoAlex** [comment]
This command seems to be not yet registered but invoked already... I'll get this fixed.

**Comment by Eskibear** [comment]
I just took a glance at the occurrences, not sure if it's related. 

https://github.com/spring-projects/sts4/blob/94dd78b2fce7def6fb5508cf4fcd4c5c9f601167/vscode-extensions/commons-vscode/src/classpath.ts#L13-L16

JDTLS might not yet be ready when Java extension is activated. Here we should probably check [javaApi.serverReady()](https://github.com/redhat-developer/vscode-java/blob/c0e0a6376809e6c1d5f60bc5a2bfd07452e9a6db/src/extension.api.ts#L114-L120) before sending requests to jdtls.

**Comment by BoykoAlex** [comment]
Should be fixed with: 3737f03e5acd30deae88c5ab1373e31a44c401ad
